spec/helpers/calendar_helper_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'when add_to_google_cal is true without a refill date' do
       medication.add_to_google_cal = true
       medication.refill = nil
-      expect(new_cal_refill_reminder_needed?(medication)).to eq(false)
+      expect(new_cal_refill_reminder_needed?(medication)).to eq(nil)
```

Comment:
Why was this changed to nil?

Reply:
so, in ruby nil is a negative value in bool expressions. for example

if nil
  true
else
  false
end

#=> false

but nil != false and this is why test is falling down with false instead of nil.

Reply:
we can also change `#new_cal_refill_reminder_needed` to this:

```
!!(medication.add_to_google_cal && medication.refill &&
  (medication.add_to_google_cal_changed? || medication.refill_changed?))
```

it will have a similar result, but I'm not sure that this is good practice.

Reply:
Hmm, ideally that method should return a boolean. Returning either true or nil isn't very consistent to me. 

Double nots are messy too, as you pointed out, what about `nil?`